content/articles/registering-domain.markdown

```diff
@@ -7,25 +7,19 @@ categories:
 
 # Registering a Domain
 
-<div class="section-steps" markdown="1">
-##### To register a new domain
+There are **two** reasons to register a domain with DNSimple: when it's a newly purchased domain, or to register a domain for which you're already managing the DNS.
 
-1.  Log in to DNSimple with your user credentials.
-1.  If you have more than one account, select the relevant one.
-1.  On the top-nav menu click the <label>Domains</label> tab.
-1.  Click <label>Register a Domain</label>.
-1.  Fill out the domain name.
-1.  Choose whether you'd like Whois Privacy Protection or auto-renewal on the domain (if applicable).
-1.  Click <label>Register Domain</label>.
-1.  Provide contact details for the registrant of the domain, or choose a contact from those you've already created.
-1.  Click <label>Register Domain</label>. You will be charged immediately for the registration.
-</div>
+When applicable, we recommend opting for our "auto-renew" feature to avoid your domain expiring if you plan to retain ownership.
 
 <div class="section-steps" markdown="1">
-##### To register a domain you're already managing the DNS for
+##### To register a new domain
```

Comment:
since this turned into a blanket section, would it make more sense to drop the 'new' and make it just "To register a domain" ?

Reply:
I'd agree. Thoughts, @SarahCaminiti1 ?

Reply:
I agree!